cmd/geth/main.go

```diff
@@ -146,6 +146,10 @@ var (
 		utils.EWASMInterpreterFlag,
 		utils.EVMInterpreterFlag,
 		configFileFlag,
+		// Additional Optimism flags
+		utils.TxIngestionEnableFlag,
+		utils.TxIngestionDBHostFlag,
+		utils.TxIngestionPollIntervalFlag,
```

Comment:
Make sure that all the flags are defined here that are used in `cmd/geth/usage.go`